model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityResult.java

```diff
@@ -33,6 +33,9 @@ public class RecordSimilarityResult {
     Map<String, RecordSimilarityField> left;
     Map<String, RecordSimilarityField> right;
     RecordSimilarityExplainInfo explainInfo;
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
are any of the other fields in this class things we want in the response if they're empty?  if not (and I suspect that's the case), this might make more sense as a class-level annotation instead of a property-level one

Reply:
I would think, "left" and "right" are probably worth showing, even if empty, but I'll apply it to "error" as well